src/AWS.Deploy.Common/Recommendation.cs

```diff
@@ -105,14 +105,30 @@ public OptionSettingItem GetOptionSetting(string jsonPath)
             return optionSetting;
         }
 
-        public T GetOptionSettingValue<T>(OptionSettingItem optionSetting, bool ignoreDefaultValue = false)
+        public T GetOptionSettingValue<T>(OptionSettingItem optionSetting, bool ignoreDefaultValue = false, bool onlyDisplayableChildren = false)

```

Comment:
Can you add tests that covers the new param.

Reply:
I added 2 tests in this PR https://github.com/aws/aws-dotnet-deploy/pull/200.
We don't have an option setting that needs this in this PR.